python/jiminy_py/src/jiminy_py/log.py

```diff
@@ -245,10 +245,11 @@ def extract_trajectory_data_from_log(log_data: Dict[str, np.ndarray],
     return traj_data
 
 
-def build_robot_from_log(
+def build_robots_from_log(
         log_constants: Dict[str, str],
-        mesh_package_dirs: Union[str, Sequence[str]] = ()) -> jiminy.Model:
-    """Build robot from log constants.
+        mesh_package_dirs: Union[str, Sequence[str]] = ()
+        ) -> Dict[str, jiminy.Model]:
```

Comment:
I'm not sure about this because the name of the function has changed (as we decided) so it already breaks the original behavior. And each time `build_robots_from_log` is called in the code I have to add `if isinstance(robots_log_file, jiminy.Robot): robots_log_file = {'': robots_log_file}`. What do you think ?